Edited Readme.md to make it easier for Windows users to install

Tbones41 opened the pull request:
Edited a few grammatical errors. 
Added information about where to place the .env file. 
Didn't edit Ubuntu installation except to include Git

Tbones41 committed:
Edited Readme.md to make it easier for Windows users to install

Tbones41 commented:
Also added a comment about Heroku, Travis CI and Ruby

Nkoli commented on README.md:
```diff
@@ -3,20 +3,22 @@
 * Python3. Run the 'python3 -V' command to see the version you have installed.
 
-* [PostgreSQL](https://www.postgresql.org/) (Ensure the server is running).
+* Git
```
It's assumed that whoever cloned this project already has Git installed. No need to explicitly add this.

Nkoli commented on README.md:
```diff
@@ -76,44 +82,51 @@ flake8 .
 
 * Download [pip](https://pip.pypa.io/en/latest/installing/) and follow the instructions in the link as an installation guide.
 
+* Download and install [Git](https://git-scm.com/download/win). Clone the project to a fresh directory/folder from the `staging` branch. 
```
Same comment as the one left above. If they clone, they get all the branches and can checkout to the available ones to see what's being done there. No need to explicitly add this.

Nkoli commented on README.md:
````diff
@@ -76,44 +82,51 @@ flake8 .
+  ```
 
-* Create a new virtual environment:
+* Then, create a new virtual environment:
````
Start the sentence with 'Create', remove 'then'

Nkoli commented on README.md:
````diff
@@ -76,44 +82,51 @@ flake8 .
+  ```
 
-* Create a database with PostgresQL, if you installed it earlier. If not, installation instructions can be found [here](https://www.postgresql.org/download/windows/)
+* Create a database with PostgresQL, if you installed it earlier. If not, installation instructions can be found [here](https://www.postgresql.org/download/windows/). Make sure to note Database name, Database Username and Password.
````
nit: lowercase characters for the db sentences and variables you added.

Nkoli commented on README.md:
```diff
@@ -76,44 +82,51 @@ flake8 .
 
-* Create a .env file and copy the content of `.env.example` file to it.
-* Replace
+* Create a .env file and copy the content of `.env.example` file to it. Place this file in the folder named `core`.
```
nit: How about editing the last sentence to _place this file in the root directory of the core project_

Nkoli commented on README.md:
```diff
@@ -152,7 +165,10 @@ flake8 .
-Clone the repository and please read the [contributing guide](/CONTRIBUTING.md).
+Clone the repository from the `staging` branch and please read the [contributing guide](/CONTRIBUTING.md).
+
+You may also need to have `Heroku`, `Ruby` and `Travis CI` installed and signed in. 
```
_Travis and Heroku command line clients_.

Also, before Travis can be installed, it's in their official documentation that certain Ruby versions need to be installed. Since the complete instructions for Ruby installation is also there, I don't think there's any point stating it here, they'll see it in the Travis documentation. Maybe remove Ruby, and also add that the Heroku and Travis login commands needs to be run in the terminal after the command line clients have been successfully installed.

Tbones41 committed:
Make corrections to readme.md

made corrections as suggested by Nkoli
removed redundancies and added login
commands for Travis and Heroku

Resolves: #51

Nkoli reviewed (approved):
LGTM
